src/baltpeter/Internetmarke/StampPngResult.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace baltpeter\Internetmarke;
+
+class StampPngResult extends ApiResult {
```

Comment:
As far as I can see, the API results for PNGs and PDFs are identical, so we could generalize this class for both of them.

Reply:
For PNG the link points to a zip file that includes 1 or more stamps. For PDF, the link just points to a single PDF that contains all stamps.

Therefore, I would return an stdClass for PDF, because there is no need to have an unzip function for the PDF return object.

Reply:
Good point. Sorry, it's been a while since I used the API.